distribution/lib/Standard/Table/0.0.0-dev/src/Errors.enso

```diff
@@ -41,7 +41,7 @@ type Too_Many_Column_Names_Provided
     ## PRIVATE
        More names than the column count provided to the function.
        Can occur when using By_Position.
-    Error (column_names : [Text])
+    Error (column_names : Vector Text)
```

Comment:
Wow that is an ancient artifact you found here!

Reply:
Yep - I was thinking about adding a check to linter for this style of type.